VideoLocker/src/main/java/org/edx/mobile/util/FileUtil.java

```diff
@@ -38,4 +41,25 @@
             inputStream.close();
         }
     }
+
+    /**
+     * Deletes a single file and/or recursively deletes files/folders within a folder before
+     * deleting it.
+     *
+     * @param fileOrDirectory The file/folder that needs to be deleted.
```

Comment:
The '/' should be replaced with 'or'. Applies to the documentation of the next param as well.

Also, 'directory' is a technically more accurate terminology than 'folder'. This applies to the whole javadoc here.